### DISCUSSION TITLE
Seq docker does not start, wrong metastore.collection.xxxxxxxxxxxxxxxx.docc

### DISCUSSION THREAD

**Discussion by lassiko** [datalust/seq-tickets#1874] [Help]
Hello, is there any way to recover:

seq does not start with this message, I can get a bash-container running, but even backup:

The backup could not be restored; an exception occurred.
Exception message: Flare native storage failed (InternalError), data file `"/data/Documents/metastore.collection.33ffdcc3783d48f4a1761e6956d2b3ef.docc"` is missing.

I have a metastore collection file, but it has a different GUID than the one seq wants..

root@601db7971e95:/data/Documents# ls
metastore.2.metadata  metastore.collection.42ff329a368f4b98b942ffd287d9a91d.docc  metastore.flare

**Comment by nblumhardt** [comment]
Hi! Thanks for getting in touch. Are you running Seq from a file share, e.g. Azure Files or a similar technology? If so, this data corruption issue may occur again in the future, and it will be a good idea to change to a different storage technology if the data is important to you.

You'll need to completely remove all files from the Documents/ directory before restoring from backup again.

(If restore fails, you may need to briefly start up/shut down the container once you've deleted the contents of Documents/ in order to initialize a new document store. Ensure the instance isn't externally accessible while you do this, as you may end up without any authentication set up, depending on how you've configured the container.)

Let me know if this helps,
Nick
